doc/scripts/spicy-doc-to-rst

```diff
@@ -296,6 +296,43 @@ class Method:
         return self.rst() < other.rst()
 
 
+# Manages a single output file.
+class OutputFile:
+    def __init__(self, ns, args):
```

Comment:
Nit: instead of passing `args` we could pass `args.dir` directly (probably as a parameter named `directory` to not hide the global `dir` function).

Reply:
Yeah, but seems fine either way, I'll leave it as is.